dev_guide/quota.adoc

```diff
@@ -29,19 +29,22 @@ The following describes the set of limits that may be enforced by a
 |Resource Name |Description
 
 |`cpu`
-|Total requested cpu usage across all containers
+|Total of cpu requests across all containers
 
 |`memory`
-|Total requested memory usage across all containers
+|Total of memory requests across all containers
 
-|`openshift.io/imagesize` [[imagesize]]<<zero,*>>
-|Maximum size of a single image stored in an internal registry
+|`requests.cpu`
+|Total of cpu requests across all containers
 
-|`openshift.io/imagestreamsize` [[imagestreamsize]]<<zero,*>>
-|Maximum size of image stream's images stored in an internal registry
+|`requests.memory`
+|Total of memory requests across all containers
 
-|`openshift.io/projectimagessize`
-|Maximum size of all the project's images stored in an internal registry
+|`limits.cpu`
+|Total of cpu limits usage across all containers
```

Comment:
Above you wrote "Total of memory requests across" but here you have "Total of cpu limits **usage** across" - do you want to be consistent and either use "usage" or not?